lib/chewy/type/import.rb

```diff
@@ -31,10 +32,13 @@ def import *args
 
           ActiveSupport::Notifications.instrument 'import_objects.chewy', type: self do |payload|
             adapter.import(*args, import_options) do |action_objects|
+              journal = Chewy::Journal.new(self)
+              journal.add(action_objects) if import_options[:journal] || journal?
```

Comment:
Still if import_options[:journal] will be false, it will not redefine the global config.
`import_options.fetch(:journal) { journal? }`